dist/redhat/scylla.spec

```diff
@@ -192,6 +195,7 @@ fi
 %if 0%{housekeeping}
 %config(noreplace) %{_sysconfdir}/scylla.d/housekeeping.cfg
 %endif
+%config %{_sysconfdir}/bash_completion.d/nodetool-completion
```

Comment:
This isn't a configuration file. If you copied it from scylla-tools-java, then we can clean it up later.

Reply:
Let me check.

Reply:
Done in v3.

Reply:
that's why we `Provides: ` this file.